spec/05-classes-and-objects.md

```diff
@@ -502,10 +502,11 @@ definition apply to all constituent definitions.  The rules governing
 the validity and meaning of a modifier are as follows.
 
 ### `private`
-The `private` modifier can be used with any definition or
-declaration in a template.  Such members can be accessed only from
+The `private` modifier can be used with any template definition or any definition or
+declaration in a template. Private members of a template can be accessed only from
 within the directly enclosing template and its companion module or
-[companion class](#object-definitions).
+[companion class](#object-definitions). Private templates that are not template
+members themselves can be accessed only from within the same package.
```

Comment:
```suggestion
[companion class](#object-definitions). Private [top-level](09-top-level-definitions.html#packagings)
templates can be accessed only from within the same package.
```

Reply:
Aha! It was already specced there:

> A packaging package p { ds } injects all definitions ds in as members into the package whose qualified name is p.
Members of a package are called top-level definitions. _If a definition in ds is labeled private, it is visible only for
other members in the package._

Still it may be good to have it here too.

Reply:
Ah, right. Here we're already narrowed down to the spec for members. So it's probably better to add a `Note` below instead of changing the actual spec.